src/ui/public/vis/editors/default/default.js

```diff
@@ -44,7 +44,9 @@ const defaultEditor = function ($rootScope, $compile) {
 
       if (!this.vis.type.editorConfig.optionTabs && this.vis.type.editorConfig.optionsTemplate) {
         this.vis.type.editorConfig.optionTabs = [
-          { name: 'options', title: 'Options', editor: this.vis.type.editorConfig.optionsTemplate }
+          { name: 'options',
```

Comment:
If you split up an object over multiple lines, please put the braces on different lines, as stated in our [style guide](https://github.com/elastic/kibana/blob/master/style_guides/js_style_guide.md#object--array-creation), e.g.:

```js
{
  name: 'options',
  title: i18n(...),
  editor: ...,
}
```